Misc/NEWS.d/next/Library/2024-05-21-20-13-23.gh-issue-118911.iG8nMq.rst

```diff
@@ -0,0 +1,2 @@
+To match IPython's behaviour, updated `maybe-accept`'s logic so that if the user hits enter twice, they are able to terminate the block even if there's trailing whitespace.
+Also, trimming trailing whitespace in HistoricalReader so that when the user hits arrow up, the cursor is on the last functional line. This matches IPython's behavior.
```

Comment:
It looks like you might have a redundant "This matches IPython's behavior." at the end of this news entry (as you already mention this at the beginning of the blurb).

Reply:
Agree with Savannah, but actually there's another thing here to consider.

We can say that "This change matches IPython's behavior" to communicate precedent in the chosen design. However, we don't want to say that we're making changes "to match IPython's behavior". This is not a design goal for us. It's too deep of a rabbit hole to commit to replicating IPython ☺️